.codesandbox/ci.json

```diff
@@ -4,6 +4,7 @@
     "github/algolia/instantsearch/tree/templates/react-instantsearch",
     "/examples/react/default-theme",
     "/examples/react/next-app-router",
+    "/examples/react/getting-started",
```

Comment:
should we replace `default-theme` with `getting-started` or just revert this change?

Reply:
I'll revert this and we can decide in another PR.